UIKitVK/UIKitVK/View/NewsHeaderTableViewCell.swift

```diff
@@ -0,0 +1,36 @@
+// NewsHeaderTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка заголовка новости
+final class NewsHeaderTableViewCell: UITableViewCell {
+    // MARK: - Private IBOutlet
+
+    @IBOutlet private var avatarImageView: UIImageView!
+    @IBOutlet private var nameLabel: UILabel!
+
+    // MARK: - Public Methods
+
+    func configure(_ news: NewsItem) {
+        let date = dateFormat(dateForm: news.date)
+        guard let name = news.creatorName else { return }
+        nameLabel.attributedText = NSMutableAttributedString()
+            .bold("\(name)\n")
+            .normal("\(date)")
+        guard let avatarURL = news.avatarURL else { return }
+        avatarImageView.loadImage(url: avatarURL)
+    }
+
+    // MARK: - Private Methods
+
+    private func dateFormat(dateForm: Double) -> String {
```

Comment:
```suggestion
    private func dateFormat(dateForm: Int) -> String {
```
Дата нам приходит в интовом типе